src/mgr.sol

```diff
@@ -186,124 +189,117 @@ contract TinlakeManager {
         z = x > y ? y : x;
     }
 
+
+    // moves the rwaToken into the vault
+    // requires that mgr contract holds the rwaToken
+    function lock(uint wad) public operatorOnly {
```

Comment:
should we simply call `urn.lock` in `join`?

Reply:
It depends. If we want to lock 100% of the rwaToken in the beginning or calculated the percentage of locked DROP (required drop to borrow up to the the debt ceiling would be 100%) and move the corresponding rwaToken amount